website/tools.shtml

```diff
@@ -69,4 +69,16 @@
     is useful to keep problem data secret and to make sharing credentials easier on the organizer.
 </p>
 
+<h3>DOMjudge for teaching</h3>
+
+<p> DOMjudge is mostly focused on being a contest system for short running contests. Some people use the
+automatic judging for teaching. When possible we can incorporate such extensions to provide easier access
+through API and feature requests. Because not everything will make it into our releases we link here to
+some extensions to help with this usecase.
```

Comment:
```suggestion
<p> DOMjudge is mostly focused on being a contest system for short running contests. Some people use the
automatic judging for teaching. When possible we can incorporate such extensions to provide easier access
through the API and UI with feature requests. Since not everything will make it into our releases, we link here to
some extensions to help with this use case.
```